packages/create-pantheon-decoupled-kit/src/generators/next-wp-search-api-addon.generator.ts

```diff
@@ -0,0 +1,40 @@
+import chalk from 'chalk';
+import {
+	addWithDiff,
+	convertCSSModules,
+	runInstall,
+	runLint,
+} from '../actions';
+import type { DecoupledKitGenerator, DefaultAnswers } from '../types';
+import { outDirPrompt, tailwindcssPrompt } from '../utils/sharedPrompts';
+
+interface NextWpSearchApiAddonAnswers {
+	search: true;
+}
```

Comment:
I moved these interfaces into `types.ts` and made a `BaseGeneratorData` type that these should extend. This way all of the data passed into the templates is type-safe. LMK if you want a more detailed walkthrough on that.